core/shared/mem-alloc/ems/ems_alloc.c

```diff
@@ -445,12 +445,27 @@ static int
 do_gc_heap(gc_heap_t *heap)
 {
     int ret = GC_SUCCESS;
+#if WASM_ENABLE_GC_PERFORMANCE != 0
+    double time = 0;
+    clock_t start = 0, end = 0;
 
+    start = clock();
+#endif
     if (heap->is_reclaim_enabled) {
         UNLOCK_HEAP(heap);
         ret = gci_gc_heap(heap);
         LOCK_HEAP(heap);
     }
+#if WASM_ENABLE_GC_PERFORMANCE != 0
+    end = clock();
+    time = end - start;
+    time = time / CLOCKS_PER_SEC;
```

Comment:
I guess the time unit here is second, right? Could we use ms or us as the unit, and change the code like:
```C
    uint32 time; (or uint64 time)
    ...
    time = time * 1000 / CLOCK_PER_SEC;  (or time = time * 1000000 / CLOCK_PER_SEC; or us)
```
And in L460, we can use `os_time_get_boot_microsecond` instead, the time unit is us.